docs/topics/native/native-objc-interop.md

```diff
@@ -228,7 +228,11 @@ A Swift/Objective-C initializer is imported to Kotlin as constructors or as fact
 The latter happens with initializers declared in the Objective-C category or as a Swift extension,
 because Kotlin has no concept of extension constructors.
 
-Kotlin constructors are imported as initializers to Swift/Objective-C. 
+> Remember to annotate Swift methods with `@objc` before importing Swift initializers to Kotlin.
```

Comment:
Technically, initializers are not methods.
So, the more precise way to say this would be:
> Remember to annotate Swift initializers with `@objc` before importing them Kotlin.